### ISSUE TITLE
Display Contract name, link and concession name on project info

### ISSUE THREAD

**Issue by davidmihalyi** [NRGI/resourceprojects.org-frontend#183]
Contract name and concession name don't show currently in project info, just place holder.
Contract name should be a link if there is a link to a contract. (open in new tab)

**Comment by davidmihalyi** [comment]
Antamina for example should have both info:
http://lodspeakr-live.nrgi-dev2.default.opendataservices.uk0.bigv.io/project/PE/anta-rzaeth.html

**Comment by caprenter** [comment]
@timdavies - do we access this info via base_def:source
http://lodspeakr-live.nrgi-dev2.default.opendataservices.uk0.bigv.io/default-view/http/resourceprojects.org/project/PE/anta-rzaeth

**Comment by timgdavies** [comment]
This looks like another model issue. I'm just updating the ontology and will push up shortly which should give us ```rp:Contract rp:relatedProject rp:Project``` triples in the data.

**Comment by timgdavies** [comment]
Ontology is updated in https://github.com/NRGI/resource-projects-etl/pull/59 and so once that is merged into ETL and data reloaded it should be possible to find the relationships to make this work.

This also required a change so that the template was #contract+project rather than #project+contract. 

(As an aside @caprenter - we should avoid using ```rp:source``` in queries for anything other than provenance. If you find you are needing to use it - let me know as probably means a data model issue)

**Comment by caprenter** [comment]
I don't think I'm using source anywhere, so should be ok

**Comment by caprenter** [comment]
I'm seeing some related concession data here (Antamina ):
http://lodspeakr-live.nrgi-dev2.default.opendataservices.uk0.bigv.io/default-view/http/resourceprojects.org/project/PE/anta-v3tsjf

But I'm not sure how to turn that into data to show in the Project Info box - I can't get to a 'name' - without going via source again?

**Comment by timgdavies** [comment]
Which source graph is this from? It might need a spreadsheet template
tweak...

**Comment by caprenter** [comment]
This is from the Antamina project in Peru, last loaded 21hrs ago
http://etl.nrgi-dev2.default.opendataservices.uk0.bigv.io/dataload/11/

**Comment by caprenter** [comment]
This may or may not be a priority for @timgdavies but just marking it so I remember it's important

**Comment by davidmihalyi** [comment]
Yes, this is a key feature, unless much work.

Sent from mobile;  pls excuse brevity and typos.

On 22 Oct 2015, at 23:14, caprenter <notifications@github.com<mailto:notifications@github.com>> wrote:


This may or may not be a priority for @timgdavies<https://github.com/timgdavies> but just marking it so I remember it's important

-

**Comment by caprenter** [comment]
Sorry. We realise this is key, but there seems to be a bug in the data model that I'm unable to fathom myself to actually get the data to populate the front end.

Again really sorry this has not made it in as yet

**Comment by caprenter** [comment]
This is now happening via Tim's commit above
